experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -72,16 +82,16 @@ def get_fml_errors(
         version: Optional[NimbusFeatureVersion] = None,
     ):
         """Fetch errors from the FML. This method creates FML clients, which are
-        used by `FmlFeatureInspector`s to fetch errors based on the blob of text and
-        the given feature.
+        used by `FmlFeatureInspector`s to fetch FML errors based on a blob of text and
+        the given feature id.
 
         Returns:
             A list of feature manifest errors.
         """
         if self.application is not None:
             errors = []
             if client := self.fml_client(version):
-                if inspector := self._get_inspectors(client, feature_id):
+                if inspector := self.feature_inspector(client, feature_id):
                     if errs := self._get_errors(inspector, blob):
```

Comment:
_get_errors is still here.

Reply:
oh oh I thought you meant the docstring 😄 removing this now